infrastructure/Pulumi.production.yaml

```diff
@@ -3,3 +3,4 @@ config:
   www.pulumi.com:certificateArn: arn:aws:acm:us-east-1:058607598222:certificate/a3f72a2b-e715-4639-b126-1e4efc0b634b
   www.pulumi.com:pathToWebsiteContents: ../public
   www.pulumi.com:targetDomain: www-production.pulumi.com
+  www.pulumi.com:redirectDomain: www.pulumi.com
```

Comment:
I assume we'd want to update the `Pulumi.staging.yaml` as well? Even if the repo isn't regularly updated, we should keep it up to date so to avoid confusing the next time we do go an update it.

Reply:
What I really want to do is just use the `targetDomain`. The way the code is written, it will use the `redirectDomain` if set, otherwise use the `targetDomain`.

```typescript
const redirectDomain = config.redirectDomain || config.targetDomain;
```

I had to add this `redirectDomain` config because the `targetDomain` for production is not the domain we actually want to use. (It is correct for staging). For production it is `www-production.pulumi.com` but we want to use `www.pulumi.com`. This is fallout of this being set outside of the Pulumi program.